tests/bson/bson-timestamp-006.phpt

```diff
@@ -0,0 +1,56 @@
+--TEST--
+MongoDB\BSON\Timestamp serialization (64-bit)
```

Comment:
On 32-bit, it's possible for Timestamp to serialize its increment or timestamp as a negative number. Such a Timestamp could not currently be created from the constructor (zpp parses as a long), but may be created by decoding valid BSON. I believe this only presents a problem if a 64-bit platform were to unserialize data from a 32-bit platform.

On a related note, 32-bit systems cannot create valid Timestamp BSON if either value exceeds 2147483647 (largest 32-bit signed integer). Should we switch to parsing arguments as strings for 32-bit, like we do with UTCDateTime? From there, we can use `STRTOLL()` and always check the valid range appropriately. It would also be easy to support cross-platform serialization (much like UTCDateTime's initialization, which can pick up long and string values from either platform).